cadquery/occ_impl/shapes.py

```diff
@@ -417,6 +417,8 @@ def exportStl(
         tolerance: float = 1e-3,
         angularTolerance: float = 0.1,
         ascii: bool = False,
+        isToleranceRelative: bool = True,
```

Comment:
Could you simply use `relative` here?

Reply:
I made it more specific in an attempt to be clear that it applies to `tolerance` only, not `angularTolerance`. I tend toward including ‘is’ in Boolean names, but I realize that’s not consistent with the rest of cadquery.

I suppose someone could also interpret `relative` to have some effect on the spatial placement of the exported shape relative to the origin.

Both of the above can be resolved by reading the docstring though. I can switch it if you like.

Reply:
+1 for `relative`

Reply:
> Both of the above can be resolved by reading the docstring though. I can switch it if you like.

yes, please

Reply:
Done ✅

Reply:
Thank you!